src/pages/ProductList/productList.js

```diff
@@ -4,18 +4,24 @@ import Filter from '../../components/filter/filter';
 import ProductCard from '../../components/productCard/productCard';
 import { products } from '../../backend/db/products';
 import { useFilter, FilterProvider } from '../../context/filterContext';
```

Comment:
FilterProvider is a context component so its not used here so not need to use it here. You have to import this in index.js file and use it there